examples/rbfe_edge_list.py

```diff
@@ -83,14 +88,14 @@ def read_from_args():
                 )
             )
 
-    for fut, meta in zip(futures, metadata):
-
+    for i, meta in enumerate(metadata):
         mol_a, mol_b, _, _, exp_ddg, fep_ddg, fep_ddg_err, ccc_ddg, ccc_ddg_err = meta
         mol_a_name = get_mol_name(mol_a)
         mol_b_name = get_mol_name(mol_b)
 
         try:
-            solvent_res, solvent_top, complex_res, complex_top = fut.result()
+            complex_res, complex_top = cfutures[i].result()
+            solvent_res, solvent_top = sfutures[i].result()
```

Comment:
oops - meant to put the comment here. might be good to get the `result()` of the solvent futures first since its faster, but it probably doesn't matter much in practice since the analysis needs both `complex` and `solvent`, maybe later on when we can decouple the windows entirely we can be more clever about this.

Reply:
Right. I'll swap in case the result for solvent throws first. Definitely can be improved later.